main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -623,49 +635,8 @@ Resources:
               - dynamodb:Scan
               - dynamodb:UpdateItem
             Resource:
-              - !GetAtt [PasswordsDb, Arn]
-              - !GetAtt [UsersDb, Arn]
-              - !GetAtt [LocksDb, Arn]
-              - !GetAtt [StepTemplatesDb, Arn]
-              - !GetAtt [WorkflowTemplatesDb, Arn]
-              - !GetAtt [WorkflowTemplateDraftsDb, Arn]
-              - !Join ['', [!GetAtt [WorkflowTemplateDraftsDb, Arn], '/index/*']]
-              - !GetAtt [WorkflowsDb, Arn]
-              - !GetAtt [WorkflowDraftsDb, Arn]
-              - !Join ['', [!GetAtt [WorkflowDraftsDb, Arn], '/index/*']]
-              - !GetAtt [WorkflowInstancesDb, Arn]
-              - !Join ['', [!GetAtt [WorkflowInstancesDb, Arn], '/index/*']]
-              - !GetAtt [WfAssignmentsDb, Arn]
-              - !Join ['', [!GetAtt [WfAssignmentsDb, Arn], '/index/*']]
-              - !GetAtt [EnvironmentsDb, Arn]
-              - !GetAtt [EnvironmentsScDb, Arn]
-              - !Join ['', [!GetAtt [EnvironmentsScDb, Arn], '/index/*']]
-              - !GetAtt [EnvironmentsTypesDb, Arn]
-              - !GetAtt [StudiesDb, Arn]
-              - !GetAtt [ProjectsDb, Arn]
-              - !GetAtt [StudyPermissionsDb, Arn]
-              - !GetAtt [IndexesDb, Arn]
-              - !GetAtt [AccountsDb, Arn]
-              - !GetAtt [AwsAccountsDb, Arn]
-              - !GetAtt [AuthenticationProviderTypesDb, Arn]
-              - !GetAtt [AuthenticationProviderConfigsDb, Arn]
-              - !GetAtt [RevokedTokensDb, Arn]
-              - !Join ['', [!GetAtt [StudiesDb, Arn], '/index/*']]
-              - !GetAtt [IndexesDb, Arn]
-              - !GetAtt [CostApiCachesDb, Arn]
-              - !GetAtt [UserRolesDb, Arn]
-              - !GetAtt [StorageGatewayDb, Arn]
-              - !GetAtt [DsAccountsDb, Arn]
-              - !GetAtt [RoleAllocationsDb, Arn]
-              - !GetAtt [ResourceUsagesDb, Arn]
-              - !If
-                - EnableEgressStore
-                - !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${self:custom.settings.dbEgressStore}'
-                - !Ref 'AWS::NoValue'
-              - !If
-                - EnableEgressStore
-                - !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${self:custom.settings.dbEgressStore}/index/*'
-                - !Ref 'AWS::NoValue'
+              - !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${self:custom.settings.dbPrefix}-*'
```

Comment:
The policy for this role was getting too long, and CFN was throwing an error. This change was made to shorten the policy length.